src/frontend/Komponenter/Personoversikt/Infotrygdperioderoversikt.tsx

```diff
@@ -12,15 +12,23 @@ import { Stønadstype } from '../../App/typer/behandlingstema';
 import SummertePerioder from '../Migrering/SummertePerioder';
 import InfotrygdPerioder from '../Migrering/InfotrygdPerioder';
 import MigrerBarnetilsyn from '../Migrering/MigrerBarnetilsyn';
-import { AlertInfo } from '../../Felles/Visningskomponenter/Alerts';
+import { AlertInfo, AlertWarning } from '../../Felles/Visningskomponenter/Alerts';
 import { Checkbox } from '@navikt/ds-react';
+import { IHistoriskPensjon } from '../../App/typer/historiskpensjon';
 
+const FlexBox = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 1rem;
+`;
 const StyledAlertStripe = styled(AlertInfo)`
-    margin: 1rem 0;
-    max-width: 51rem;
-    .navds-alert__wrapper {
-        max-width: 51rem;
-    }
+    width: 40rem;
+`;
```

Comment:
Hva er grunnen til at den ble smalere og andre delene fra StyledAlertStripe ble fjernet? :)

Reply:
Mest for at de skulle få lik lengde ved siden av hverandre, og ikke være for brede